index.html

```diff
@@ -113,8 +113,12 @@
       </div>
       <div class="col-md-7">
         <h2>What <em>is</em> Standard?</h2>
-        <p>Standard is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're
-        released and drop out about eighteen months later.</p>
+        <p>
+          Standard is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're
+          released and drop out about eighteen months later. Some sets are labeled as <strong>Non-standard</strong> or <strong>Modern</strong> and are thus not
+          included in this group. <a class="tip" title="For example, Reprint/Masters sets are Modern sets, not standard sets."></a>
+          The sets that <i>are standard</i> are <a href="http://magic.wizards.com/en/content/standard-formats-magic-gathering">listed here.</a>
+        </p>
```

Comment:
I am in favor of the fix, but I'd prefer to shave this down. imo this part

> Reprint/Masters sets are Modern sets, not standard sets.

is the key, and the rest can fall off.

(Also remember that Standard is capitalized)